private/bufpkg/bufmodule/util.go

```diff
@@ -72,6 +72,20 @@ func getDocumentationForBucket(
 	return string(documentationData), nil
 }
 
+func getLicenseForBucket(
```

Comment:
This isn't materially different from `getDocumentationForBucket`.  Maybe it's time to put this logic as `getFileForBucket` and pass the doc and license file paths to it in `get(Documentation|License)ForBucket`.